wf-federated/Snakefile

```diff
@@ -0,0 +1,44 @@
+configfile: "/data/config.yaml"
```

Comment:
So this is what Snakemake will fall back to if the config file location is not explicitly passed, correct? But using an absolute path here will almost never work. Shouldn't we use a default that points to the `config.yaml` file we ship with this repo, i.e., `configfile: "./config.yaml` in this case?

Reply:
This is a remnant from the tutorial I gave. There I put the config.yaml in /data. 
adjusted